api/db-cdr/build.gradle

```diff
@@ -14,7 +14,7 @@ def liquibase_password = System.getenv("LIQUIBASE_DB_PASSWORD") ?: "lb-notasecre
 applicationDefaultJvmArgs = ["-Xmx2048m","-Xms2048m"]
 
 dependencies {
-    liquibaseRuntime 'org.liquibase:liquibase-core:4.12.0'
```

Comment:
Looking for Gradle/Groovy help: how can we keep this aligned with the `api/build.gradle` version number?  Ideally I'd like to use the existing `LIQUIBASE_VERSION` constant